README.md

```diff
@@ -193,13 +322,149 @@ generate RBIs for dynamic methods
 ```
 <!-- END_HELP_COMMAND_DSL -->
 
-## Configuration
+#### Keeping RBI files for DSLs up-to-date
+
+To ensure all RBI files for DSLs are up-to-date with the latest changes in your application or database, Tapioca provide a `--verify` option:
+
+```sh
+$ bin/tapioca dsl --verify
+
+Loading Rails application... Done
+Loading DSL compiler classes... Done
+Checking for out-of-date RBIs...
+
+
+RBI files are out-of-date. In your development environment, please run:
+  `bin/tapioca dsl`
+Once it is complete, be sure to commit and push any changes
+
+Reason:
+  File(s) changed:
+  - sorbet/rbi/dsl/my_model.rbi
+```
+
+This option can be used on CI to make sure the RBI files are always up-to-date and ensure accurate type checking.
+
+#### Writing custom DSL compilers
+
+**TODO**
+
+### RBI files for missing constants and methods
+
+Even after generating the RBIs, it is possible that some constants or methods are still undefined for Sorbet.
+
+This might be for multiple reasons, the most frequents being:
+
+* The constant or method comes from a part of the gem that Tapioca cannot load (optional dependency, wrong architecture, etc.)
+* The constant or method comes from a DSL or meta-programming that Tapioca doesn't support yet
+* The constant or method only exists when a specific code path is executed
+
+The best way to deal with such occurrences is to manually create RBI files (shims) for them so you can also add types but depending on the amount of meta-programming used in your project this can mean an overwhelming amount of manual work.
+
+#### Generating the RBI file for missing constants
+
+To get you started quickly, Tapioca can create a RBI file containing a stub of all the missing constants so you can typecheck your project without missing constants and shim them later as you need them.
+
+To generate the RBI file for the missing constants used in your application run the following command:
+
+```sh
+$ bin/tapioca todo
+
+Compiling sorbet/rbi/todo.rbi, this may take a few seconds... Done
+All unresolved constants have been written to sorbet/rbi/todo.rbi.
+Please review changes and commit them.
+```
+
+This will generate the file `sorbet/rbi/todo.rbi` defining all unresolved constants as empty modules. This file should be reviewed then committed in your repository.
```

Comment:
```suggestion
This will generate the file `sorbet/rbi/todo.rbi` defining all unresolved constants as empty modules. Since the constants are "missing", Tapioca does not know if they should be marked as modules or classes and will use modules as a safer default. This file should be reviewed, corrected, if necessary, and then committed in your repository.
```